include/boost/dll/detail/demangling/itanium.hpp

```diff
@@ -184,7 +184,7 @@ std::string mangled_storage_impl::get_mem_fn(const std::string &name) const
              '(' + parser::arg_list(*this, func_type()) + ')'
              + const_rule<Class>() + volatile_rule<Class>();
 
-    auto found = std::find_if(storage_.begin(), storage_.end(), [&](const entry& e) {return e.demangled == matcher;});
+    auto found = std::find_if(storage_.begin(), storage_.end(), [&](const entry& e) {return e.demangled.find( matcher ) != std::string::npos;});
```

Comment:
Please add a comment `// Linux export table contains int MyClass::Func<float>(), but expected in import_mangled MyClass::Func<float>() without returned type.`

The fix could give incorrect results for Other::MyClass::Func<float>() and MyClass::Func<float>(). Looks like the check should be close to the following `e.demangled == matcher || e.demangled.find(" " + matcher) != std::string::npos;`